src/platform/OpenThread/OpenThreadUtils.cpp

```diff
@@ -130,6 +130,7 @@ void LogOpenThreadStateChange(otInstance * otInst, uint32_t flags)
             ChipLogDetail(DeviceLayer, "   Mesh Prefix: %s/64", strBuf);
         }
 #if CHIP_CONFIG_SECURITY_TEST_MODE
+
```

Comment:
Nit: needless blank line

Reply:
The change-set I had to work around this issue as a bit off what was accepted in an earlier PR, I'll remove this.